examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb

```diff
@@ -432,7 +442,7 @@
    "source": [
     "## Load Documents into the Index\n",
     "\n",
-    "Now that we have a search index, we can load documents into it. We will use the same documents we used in the previous examples. In Redis, either the Hash or JSON (if using RedisJSON in addition to RediSearch) data types can be used to store documents. We will use the HASH data type in this example. The below cells will show how to load documents into the index."
+    "Now that we have a search index, we can load documents into it. We will use the same documents we used in the previous examples. In Redis, either the HASH or JSON (if using RedisJSON in addition to RediSearch) data types can be used to store documents. We will use the HASH data type in this example. The below cells will show how to load documents into the index."
```

Comment:
'hash' appears to be the correct Redis spelling: https://redis.io/docs/data-types/

Reply:
Thank you for your input. Actually, here I want to keep the case consistent, as you can see the same word is all in uppercase letters in this line.